requirements.txt

```diff
@@ -5,6 +5,7 @@
 pbr>=0.8.2,<1.0
```

Comment:
Unrelated side note: we need to fix things to use pbr>=1.0, 0.8.2 is ancient by the current standards. It's incompatible with the remaining openstack.